Civi.php

```diff
@@ -95,6 +95,7 @@ public static function service($id) {
   public static function reset() {
     self::$statics = array();
     Civi\Core\Container::singleton();
+    \Civi::service('settings_manager')->useDefaults();
```

Comment:
I find the fact that 
\CRM_Core_Config::singleton(TRUE, TRUE); 
instantiates the system desparately confusing!

does it need to be called twice above?